packages/product/src/loaders/container.ts

```diff
@@ -94,13 +92,8 @@ export default async ({
         update: ProductOptionValueServiceTypes.UpdateProductOptionValueDTO
       }
     >(ProductOptionValue)
-  const defaultProductTagRepository = DALUtils.mikroOrmBaseRepositoryFactory<
-    ProductTag,
-    {
-      create: CreateProductTagDTO
-      update: UpdateProductTagDTO
-    }
-  >(ProductTag)
+  const defaultProductTagRepository =
```

Comment:
we could automatise that by looking at the existing repos and compare to the entity and build the missing ones